components/atom/badge/src/index.scss

```diff
@@ -53,9 +53,40 @@ $badge-transparent: (
   primary: $c-primary
 ) !default;
 
+$badge-soft: (
+  success: (
+    bgc: $c-success-light-4,
+    c: $c-success
+  ),
+  error: (
+    bgc: $c-error-light-4,
+    c: $c-error
+  ),
+  alert: (
+    bgc: $c-alert-light-4,
+    c: $c-alert
+  ),
+  info: (
+    bgc: $c-gray-light-4,
+    c: $c-gray
+  ),
+  new: (
+    bgc: $c-accent-light-4,
+    c: $c-accent
+  ),
+  neutral: (
+    bgc: $c-gray-dark-3,
```

Comment:
please change neutral values to
bgc: $c-gray-light-4,
c: $c-gray

will be the same as info, but it's ok
thanks

Reply:
When the color is reviewed, it can be integrated

Reply:
Done!